src/main/resources/db-scripts/clickhouse/clickhouse.sql

```diff
@@ -277,9 +277,52 @@ FROM clinical_event ce
          INNER JOIN patient p ON ce.patient_id = p.internal_id
          INNER JOIN cancer_study cs ON p.cancer_study_id = cs.cancer_study_id;
 
+CREATE TABLE IF NOT EXISTS genetic_alteration_cna_derived
+(
+    sample_unique_id String,
+    cancer_study_identifier LowCardinality(String),
+    hugo_gene_symbol String,
+    profile_type LowCardinality(String),
+    alteration_value Nullable(Float32)
+)
+    ENGINE = MergeTree()
+        PARTITION BY (profile_type, cancer_study_identifier)
+        ORDER BY (sample_unique_id, hugo_gene_symbol);
```

Comment:
```suggestion
        PRIMARY KEY BY (profile_type, cancer_study_identifier, hugo_gene_symbol)
        ORDER BY (profile_type, cancer_study_identifier, hugo_gene_symbol, sample_unique_id);
```

PARTITION BY creates new files based on that tuple you have passed

Reply:
Got it. I'll change to ORDER BY